do_cmake.sh

```diff
@@ -6,24 +6,30 @@ if test -e build; then
 fi
 
 PYBUILD="2"
-source /etc/os-release
-case "$ID" in
-    fedora)
-        if [ "$VERSION_ID" -ge "29" ] ; then
-            PYBUILD="3"
-        fi
-        ;;
-    rhel|centos)
-        MAJOR_VER=$(echo "$VERSION_ID" | sed -e 's/\..*$//')
-        if [ "$MAJOR_VER" -ge "8" ] ; then
-            PYBUILD="3"
-        fi
-        ;;
-    opensuse*|suse|sles)
-        PYBUILD="3"
-        WITH_RADOSGW_AMQP_ENDPOINT="OFF"
-        ;;
-esac
+if [ `uname` != FreeBSD ]; then
```

Comment:
Better would be:

````
if [ -r /etc/os-release ]; then
...
elif [ "$(uname)" == FreeBSD ]; then
else
  echo Unknown release
  exit 1
fi
````

Reply:
:+1: to this

I greatly prefer the `$(...)` form over tildes.

Reply:
@batrick 
Will update, good suggestion
@smithfarm 
Actually I prefer the backticks, because they also work in /bin/sh, and thus a bit more compatible.
And uptill now I've always used this format, but if you feel strong about this, I will rewrite.

Reply:
The issue with backticks is that if the output expands to more than one word, then the test command `[ ... ]` will emit an error. IIRC, you cannot use backtick form within quotes so I recommend using `$()`.

Reply:
Also, `$()` is standard according to the 2008 document (and probably earlier) on the `sh` language: http://pubs.opengroup.org/onlinepubs/9699919799.2008edition/

Reply:
i tested with dash. it works.